.github/workflows/add-label-on-failure-report.yml

```diff
@@ -8,9 +7,11 @@ on:
 jobs:
   add_label:
     runs-on: ubuntu-latest
-    if: ${{ contains(github.event.issue.title, 'Test failure') && !contains(github.event.issue.labels.*.name, 'needs triage') }}
+    if: ${{ contains(github.event.issue.title, 'Test failure') && !contains(github.event.issue.labels.*.name, 'needs triage') && !contains(github.event.issue.labels.*.name, 'release testing') }}
```

Comment:
If I'm reading the conditional correctly, this means that if the issue contains only one of 'needs triage' or 'release testing', it will *not* add the one that's missing.  Is this intentional?

Reply:
Yes. But I can rewrite so it adds the missing label. Also, I updated the test failure issue template so it includes both labels by default: https://github.com/openedx/wg-build-test-release/pull/349/commits/dc1e84a8f45b0be1e537eb619ebc0aec1460a12d. 

So now I'm thinking, do we need this job? I added it in case someone removed the labels when opening a new issue or an issue was independently created without using the templates report.

Reply:
I'm all for simplication, so my instinct is to say we should remove it... but, on the other hand, whoever added it in the first place presumably ran into a situation that made it desirable. 🤷🏼‍♂️ 

If we're keeping it, though, I think it would be a good idea to add the missing label if only one is present.

Reply:
Do we still need the job if we're using that as default? I'm guessing we do for people without enough permissions to add labels? I'm not familiar with the report view for people without any permissions.

Reply:
Yes, I think we need it. Here's the discussion we had back then: https://github.com/openedx/wg-build-test-release/pull/281#discussion_r1212145567

Reply:
Done: https://github.com/openedx/wg-build-test-release/pull/349/commits/50f9573a133d2dab18b4cb4983490a40f4d758cc. Thanks for the feedback, @arbrandes!